ui/app/adt/models/wardDetails.js

```diff
@@ -4,7 +4,7 @@ Bahmni.ADT.WardDetails = {};
 
 Bahmni.ADT.WardDetails.create = function (details, diagnosisStatus) {
     var detailsMap = {};
-    var attributesToCopy = ["Bed", "Ward", "Id", "Name", "Age", "Gender", "District", "Village", "Admission By", "Admission Time", "Disposition By", "Disposition Time", "ADT Notes"];
```

Comment:
@Phanindra-tw can you check changing the relevant sql query would bring the kid value instead of adding in the headers list ?

Reply:
@binduak the variable "attributesToCopy" will not be used to query from backend , In the backend response if we have kid as attribute it will copy otherwise it will be as undefined

Reply:
The backend is sending the kid attribute we added the sql statements for the same to return the "kid" attribute, the frontend is only accepting the keys mentioned in `attributesToCopy` array. So we need to add "kid" here.